src/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.cs

```diff
@@ -198,11 +198,8 @@ public sealed partial class OptionTextAttribute : System.Attribute
     public sealed partial class ProjectData
     {
         internal ProjectData() { }
-        [System.Security.SecuritySafeCriticalAttribute]
```

Comment:
There are a couple differences related to inheritance documented at https://msdn.microsoft.com/en-us/library/dd233102(v=vs.110).aspx